index.js

```diff
@@ -28,6 +28,25 @@ app.post('/add/:quizName/:count', urlencodedParser, async (req, res) => {
     res.redirect()
```

Comment:
This res.redirect() is a crash point.

A handler sends one and only one response.

res.send(...) sends the response page.
-or-
res.redirect(...) specifies a different URL to go to.

One or the other, not both, and res.redirect() with no URL will crash. Point to it in VSCode and the help will pop up and show you the required parameter.

So this res.redirect() makes no sense - go ahead and remove it.